src/SFA.DAS.Provider.PR.Web/Models/EmployerDetailsViewModel.cs

```diff
@@ -44,7 +45,7 @@ public class EmployerDetailsViewModel
 
     public string LastActionText { get; set; } = null!;
 
-    public string LastRequestType { get; set; } = null!;
+    public bool ShowAgreementId { get; set; } = true;
```

Comment:
What is the point of this boolean? if this is empty it will not show on the view right?

Reply:
More than just the agreement id is shown if the agreement id is present, so without this bool, if there is no agreement id we'll be left with a subtitle with no value associated with it